lib/tasks/travis.rake

```diff
@@ -0,0 +1,11 @@
+require 'system_commands'
+
+namespace :travis do
+  desc "Update vendor/bundle files to gem cache on tmc-server"
+  task :gemcache do
+    puts "Pushing new gems from local gem cache (vendor/bundle) to travis cache"
+    SystemCommands.sh!("find vendor/bundle -iname '*.gem'  -print0  | rsync -0 --checksum -avz  --files-from=- . web01:/srv/www/testmycode.net/travis/rubygems-cache/", {escape: false})
+    puts "Done"
+  end
+end
```

Comment:
I had initially thought that it would be done more manually and not necessarily locally... But creating that task might be worth it.